code/game/atoms.dm

```diff
@@ -622,7 +622,7 @@ its easier to just keep the beam vertical.
 
 //This proc is called when objects are created during the round by players.
 //This allows them to behave differently from objects that are mapped in, adminspawned, or purchased
-/atom/proc/Created()
+/atom/proc/Created(var/mob/user)
```

Comment:
`var/` is unnecessary in proc declarations, see contribution guidelines. I wouldn't delay the PR over it, just keep that in mind for the future.